src/workerd/io/worker.c++

```diff
@@ -1025,6 +1025,12 @@ Worker::Isolate::Isolate(kj::Own<Api> apiParam,
         }
         KJ_LOG(INFO, "console warning", message);
       });
+      lock->setErrorReporterCallback([this](jsg::Lock& js, kj::StringPtr message) {
+        if (impl->inspector != kj::none) {
+          logMessage(js, static_cast<uint16_t>(cdp::LogType::ERROR), message);
+        }
+        KJ_LOG(INFO, "unhandled error", message);
```

Comment:
Need to think about this a bit. When the error is reported to the inspector, the stack is properly reported. But the stack is dropped from the `KJ_LOG` line. Is that ok? Probably not. what else can we do here?